src/main/java/net/authorize/util/XmlUtility.java

```diff
@@ -23,6 +24,9 @@
 public final class XmlUtility {
 	private static Log logger = LogFactory.getLog(XmlUtility.class);
 	private static final String XmlHeader = "<?xml version=\"1.0\" encoding=\"UTF-8\" standalone=\"yes\"?>";
+    private static JAXBContext request_ctx = null;
+    private static JAXBContext response_ctx = null;
+    private static HashMap<String, JAXBContext> jaxbContext = new HashMap<String, JAXBContext>();
 

```

Comment:
Apart from logging, httpPost.setEntity function call is also using the xmlRequest string in HttpUtility,java